includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -874,6 +883,15 @@ public function should_show_ece_on_checkout_page() {
 		);
 	}
 
+	/**
+	 * Detects if the current page is a One Page Checkout page.
+	 *
+	 * @return boolean True if on a One Page Checkout page, false otherwise.
+	 */
+	public function is_one_page_checkout() {
```

Comment:
Nit: can we specify the return type explicitly?
```suggestion
	public function is_one_page_checkout(): bool {
```

Reply:
Thanks! Added in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4757/commits/ff597e2d29880ca2564c37968c0d0675258e8c2b